cmd/devtools/reset.go

```diff
@@ -0,0 +1,194 @@
+package devtools
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/astria/astria-cli-go/cmd"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// resetCmd represents the root reset command
+var resetCmd = &cobra.Command{
+	Use:    "reset",
+	Short:  "The root command for resetting the local development instance data.",
+	Long:   `The root command for resetting the local development instance data. The specified data will be reset to its initial state as though initialization was just run.`,
+	PreRun: cmd.SetLogLevel,
+}
+
+func init() {
+	// top level command
+	devCmd.AddCommand(resetCmd)
+	resetCmd.PersistentFlags().StringP("instance", "i", DefaultInstanceName, "Choose the target instance for resetting.")
+
+	// subcommands
+	resetCmd.AddCommand(resetConfigCmd)
+	resetCmd.AddCommand(resetEnvCmd)
+	resetCmd.AddCommand(resetStateCmd)
+
+	// flags for resetting specific env files
+	resetEnvCmd.Flags().Bool("local", false, "Reset the local environment file.")
+	resetEnvCmd.Flags().Bool("remote", false, "Reset the remote environment file.")
+	resetEnvCmd.MarkFlagsMutuallyExclusive("local", "remote")
+}
+
+// resetConfigCmd represents the 'reset config' command
+var resetConfigCmd = &cobra.Command{
+	Use:    "config",
+	Short:  "Reset the Cometbft config files.",
+	Long:   "Reset the Cometbft config files. This will return the config files to their default state as though initially created.",
+	PreRun: cmd.SetLogLevel,
+	Run:    resetConfigCmdHandler,
+}
+
+func resetConfigCmdHandler(c *cobra.Command, _ []string) {
+	// Get the instance name from the -i flag or use the default
+	instance, _ := c.Parent().PersistentFlags().GetString("instance")
+	IsInstanceNameValidOrPanic(instance)
+
+	homePath, err := os.UserHomeDir()
+	if err != nil {
+		log.WithError(err).Error("Error getting home dir")
+		panic(err)
+	}
+	localConfigDir := filepath.Join(homePath, ".astria", instance, LocalConfigDirName)
+
+	log.Infof("Resetting config for instance '%s'", instance)
+
+	// Remove the config files
+	err = os.Remove(filepath.Join(localConfigDir, DefaultCometbftGenesisFilename))
+	if err != nil {
+		fmt.Println("Error removing file:", err)
+		return
+	}
+	err = os.Remove(filepath.Join(localConfigDir, DefaultCometbftValidatorFilename))
+	if err != nil {
+		fmt.Println("Error removing file:", err)
+		return
+	}
+
+	recreateCometbftAndSequencerGenesisData(localConfigDir)
```

Comment:
We should reorganize where this function lives now that it's shared.

Maybe we move all the embedded and config file functionality to its own file in `cmd/devtools`, maybe `confighelpers.go` or something?

Reply:
Tangential, but I noticed a few misspellings, `instancDir`, that you can fix when you move things, but I can't comment on that section of the code.